cloudformation/fsx-substack.cfn.json

```diff
@@ -212,6 +263,15 @@
   },
   "Resources": {
     "FileSystem": {
+      "Metadata": {
+        "cfn-lint": {
+          "config": {
+            "ignore_checks": [
+              "E3002"
```

Comment:
Why do we need to set this?

Reply:
The reason is in my previous comment: https://github.com/aws/aws-parallelcluster/pull/1849#issuecomment-649861460